analysis/xdata/flu_workflow.json

```diff
@@ -0,0 +1,36 @@
+{
+  "mode": "workflow",
+  "inputs": [],
+  "outputs": [
+    {
+      "name": "data",
+      "type": "table",
+      "format": "csv"
+    }
+  ],
+  "steps": [
+    {
+      "name": 1,
+      "task": {
+        "inputs": [],
+        "outputs": [
+          {
+            "type": "table",
+            "name": "data",
+            "format": "csv"
+          }
+        ],
+        "mode": "python",
+        "name": "Flu trends",
+        "script_uri": "file://flu.py"
```

Comment:
Yes, are you suggesting we add this for clarity here or suggesting that URIs like `file://flu.py` should be made to not function?